patch-image.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/bash
+set -ex
+
+# PATCH argument is in the form "project refs" where refs is a list of
+# comma-separated gerrit refs, fo example refs/changes/67/759567/1
+PROJ_NAME=$(echo $1 | cut -d " " -f1)
+REFSPECS=$(echo $1 | cut -d " " -f2)
+
+cd /tmp
+git clone "https://opendev.org/openstack/$PROJ_NAME"
```

Comment:
I wonder if we should allow passing the full clone URL, so that not only upstream OpenStack patches can be tested.